plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java

```diff
@@ -156,4 +163,32 @@ private void assertPropertyDefined(String name) {
     }
   }
 
+  @Test
+  public final void testPropertiesDefinedInManifestMf() throws IOException {
+    boolean localizedMessageExists = false;
+    boolean bundleLocalizationDefined = false;
+
+    Attributes attributes = getManifestAttributes();
+    for (Entry<?, ?> entry : attributes.entrySet()) {
+      String key = entry.getKey().toString();
+      String value = entry.getValue().toString();
+      assertPropertyDefined(value);
+
+      localizedMessageExists |= value.startsWith("%");
```

Comment:
This is hard to follow. Is there a more obvious, less clever way to write this test?

Reply:
I'll unroll this into a regular `if` statement.